ui/src/ReactCustomHooks/useKafkaVertxWebSocket/useKafkaVertxWebSocket.hook.js

```diff
@@ -0,0 +1,140 @@
+import { useState, useRef, useEffect, useCallback } from 'react';
```

Comment:
To my knowledge, there are no restrictions on when hooks are used and called, other than they must always been in the same order/sequence. It does mean you can get a few layers (like in this case). I dont mind adding the linter you mention below though - just in case there is an issue here!